dashboard/scripts/delete_groups_of_user_progress.rb

```diff
@@ -0,0 +1,31 @@
+## TODO: Give this file execute permissions.
+##!/usr/bin/env ruby
+
+require_relative '../../config/environment'
+
+# This script was developed to allow us to delete progress for multiple users quickly.
+# Prior to the development of this script, it was a manual process for our Customer Success team
+# Depending on where this goes, tests should be added as deleting progress is not undoable
+def delete_all_progress_for_multiple_usernames(teacher_id, usernames)
+  follower_ids = teacher_id.followers.pluck(:student_user_id)
+  # Fetch user IDs based on usernames
+  user_ids = User.where(username: usernames).pluck(:id)
+
+  user_ids.each do |user_id|
+    next unless follower_ids.include?(student.id)
```

Comment:
Have you tested this script on a local environment? This variable `student` was never set to anything, so I'd expect the script to perhaps have an exception trying to access `id` of `nil`. Does that match with what you've seen?